src/i18n/en.yaml

```
@@ -32,6 +33,7 @@ navigation:
   subscription: Subscription
 note:
   contents: Contents
+  noEntries: No notes available
```

Same applies to other languages.
```suggestion
  empty: No notes available
```